slapd-cli

```diff
@@ -2066,6 +2075,7 @@ case $1 in
 	;;
 	backup)
 	backup
+	removeoldbackups
```

Comment:
It could be nice to also have dedicated tasks for removing backups?

Reply:
My wish was to call the purge directly when calling the backup methods

Reply:
I also think it's a good idea to call the purge directly when doing the backup.

I just mentioned we could also provide a separate target for doing only the purge. It's opened to discussion, I don't have a real use case in mind. But it does not seem a hard work.